daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -562,6 +562,10 @@ object Speedy {
         compiledPackages: CompiledPackages,
         transactionSeed: crypto.Hash,
         scenario: SExpr,
+        supportedValueVersions: VersionRange[value.ValueVersion] =
+          value.ValueVersions.DefaultSupportedVersions,
```

Comment:
Just to confirm my understanding: The default still includes the dev versions?

Reply:
You are right. I rename the constant to 'SupportedDevVersions'